src/components/popup/details/purposes/purposes.test.js

```diff
@@ -150,9 +150,9 @@ describe('Purposes and Features', () => {
 		purposes.handleSelectPublisherLegitimateInterest({isSelected: false, dataId: 2});
 
 
-		expect(selectPublisherLegitimateInterests.mock.calls[0][0]).to.equal(2);
+		expect(selectPublisherLegitimateInterests.mock.calls[0][0]).to.equal(1);
```

Comment:
Can you add test case for new "feature" ?

Reply:
✅